src/components/ArmyBuildPane.jsx

```diff
@@ -13,12 +16,34 @@ const Container = styled.div`
     margin: 30px
 `;
 
-const ArmyBuildPane = () => (
-  <Container>
-    <UsageBar />
-    <SelectionPane />
-    <ArmyList />
-  </Container>
-);
+const ArmyBuildPane = () => {
+  const [unitList, setUnitList] = useState({});
+  const [listIndex, setListIndex] = useState(1);
+
+  const addUnit = unit => {
+    setUnitList({
+      ...unitList,
+      [listIndex]: unit
+    });
+    setListIndex(listIndex + 1);
+  };
+  const removeUnit = index => {
+    const newList = { ...unitList };
+    delete newList[index];
+    setUnitList({
+      newList
+    });
+  };
+  return (
+    <Container>
+      <UsageBar usage={calculateUsage(unitList)} />
```

Comment:
having done this, wondering if it might be simpler and more efficient to just update a usage state when adding/removing 🤔

Reply:
👍 Probably!

Reply:
I guess only thing, is it's a bit more prone to error, if I call this method I'm sure the usage is based on current selections